src/aditiGJUN20/FindMaxAtDifferIndex.java

```diff
@@ -0,0 +1,35 @@
+/*
+Find max value at differed index.
+	i/p:- arr1: {10,2,9,14,3}
+	arr2: {10,2,18,14,3}
+	o/p :- Values are not matching at index -> 2, From (9,18) max value is 18 
+ * */
+
+package aditiGJUN20;
+
+public class FindMaxAtDifferIndex {
+	void getMaxAtDifferIndex(int[] arr1, int[] arr2) {
+		int max=0;
```

Comment:
No need to initialize here. You should directly initialize at line number 15.

Reply:
Made changes. 
Initialized the variable where I'm assigning the value.